README.rst

```diff
@@ -471,6 +471,23 @@ List mass upgrade operations
 
     GET /api/v1/firmware-upgrader/batch-upgrade-operation/
 
+**Available filters**
+
+You can filter a list of batch upgrade operations
+based on their build using the ``build_id``.
```

Comment:
It's not `build_id`, it's `build`.
I wouldn't use e.g. to list the possible values of the filter, because e.g. means "example", while here we're listing the possible values.

I would rewrite it as follows:

Bacth upgrade operations can be filtered by build ID and status (one of: idle, in-progress, success, failed):

.. code-block:: text

    GET /api/v1/firmware-upgrader/batch-upgrade-operation/?build={build_id}
    GET /api/v1/firmware-upgrader/batch-upgrade-operation/?status={status}

Please let's make this docs text less verbose, apply the same suggestion to all other sections.

Reply:
Sure.